docs/operator-manual/declarative-setup.md

```diff
@@ -543,6 +543,22 @@ The `resource.exclusions` node is a list of objects. Each object can have:
 
 If all three match, then the resource is ignored.
 
+In addition to exclusions, you might configure the list of included resources using the `resource.inclusions` setting.
+By default, all resource group/kinds are included. The `resource.inclusions` setting allows customizing the list of included group/kinds: 
```

Comment:
Is `resource.inclusions` mutually exclusive to the exclusions? Or is it combinable? It's not very clear to me.